pna.p4

```diff
@@ -444,7 +444,7 @@ enum PNA_CounterType_t {
 
 extern Counter<W, S> {
   Counter(bit<32> n_counters, PNA_CounterType_t type);
-  void count(in S index);
+  void count(in S index, @optional in bit<32> increment);
```

Comment:
Also should we look into adding a new method instead of modifying existing one ?

Reply:
Yes.